lndr.admin.inc

```diff
@@ -1,48 +1,66 @@
 <?php
+
 /**
  * @file
- * Code for the Lndr module
+ * Code for the Lndr administration pages.
  */
 
-
-
 /**
- * Form builder callback
+ * Form builder callback.
  */
 function lndr_admin_settings($form, &$form_state) {
-
   $form['lndr_token'] = array(
     '#type' => 'textfield',
     '#title' => t('API token'),
     '#default_value' => variable_get('lndr_token', ''),
-    '#description'   => t("Your Lndr account API token, you can find this information under your user profile in Lndr"),
+    '#description' => t("Your Lndr account API token, you can find this information under your user profile in Lndr"),
     '#required' => TRUE,
   );
 
-  $form['lndr_debug_mode'] = array(
-    '#type' => 'checkbox',
-    '#title' => t('Developer mode'),
-    '#default_value' => variable_get('lndr_debug_mode', 0),
-    '#description' => t('Turn this on to test Lndr API data from local source instead of executing real API call'),
+  if (module_exists('lndr_debug')) {
+    $form['lndr_debug_mode'] = array(
+      '#type' => 'checkbox',
+      '#title' => t('Developer mode'),
+      '#default_value' => variable_get('lndr_debug_mode', 0),
+      '#description' => t('Turn this on to test Lndr API data from local source instead of executing real API call'),
+    );
+  }
+  $desc = <<<EOF
```

Comment:
Let's make the description more clear:
Content import mode
Option 1: Content only (We will only import the index.html file while loading images and other assets from the Lndr server - fastest)
Option 2: All content and assets (All of the Lndr page content and assets from the Lndr server will be imported to the Drupal website in your public file directory)